README.md

```diff
@@ -1,3 +1,41 @@
-# itirod Work №2 "MockUp" 
+# Work №2 "MockUp" 
 
-###The first part of MockUp is [here](https://www.figma.com/file/eM6LuzvfA167lOVRHOOwAI/Untitled?node-id=0%3A1)
+
+I made the MockUp in Figma and the free version allows u to create 3 projects of 3 pages, i'm so sorry for such inconvenience :( but i hope it's not such a big problem
```

Comment:
It's okay, in the future, u can just place ur pages in one, it will be ok for a free version.